### PR TITLE
Some minor cleanup

### PR DESCRIPTION
**Pull request by vivekmenezes**

### CODE REVIEW

File: multiraft/multiraft.go [hunk 1]
```diff
@@ -348,13 +345,12 @@ func (m *MultiRaft) ChangeGroupMembership(groupID uint64, commandID string,
...
 		fn: func() {
-			err := m.multiNode.ProposeConfChange(context.Background(), uint64(groupID),
+			if err := m.multiNode.ProposeConfChange(context.Background(), uint64(groupID),
 				raftpb.ConfChange{
 					Type:    changeType,
 					NodeID:  uint64(nodeID),
 					Context: encodeCommand(commandID, payload),
-				})
-			if err != nil {
+				}); err != nil {
```
Comment (tamird) [inline multiraft/multiraft.go]: i think this is over indented?

Comment (tamird) [inline multiraft/multiraft.go]: oh, maybe the `)` needs to be on the next line

**Comment by tamird** [comment]
LGTM

File: multiraft/multiraft.go (hunk 1 above)
Comment (vivekmenezes) [inline multiraft/multiraft.go]: I'm leaving it as is. Perhaps we shouldn't be using the if err := ...; err != nil { idiom when it occupies more than a line.

**Pull request merged**